regulations/uitests/diff_test.py

```diff
@@ -74,8 +74,8 @@ def test_diffs(self):
         # wait until 1005.3 diff loads
         WebDriverWait(self.driver, 30).until(
             lambda driver:
-            driver.current_url == self.test_url
```

Comment:
Good call. We aren't using all of the flake8 plugins in -parser or omb-eregs here, but maybe we should.